<?xml version="1.0"?>

<launch>

   <node name="unification_executor" pkg="unification_roscontrol" type="unification_executor.py">
  </node>

  <include file="$(find unification_roscontrol)/launch/unification_drivers.launch" />

  <include file="$(find unification_roscontrol)/launch/scene_master.launch" />

</launch>